CommandHelper/LocalPackages/commands/buffs/greninja.ms

```diff
@@ -0,0 +1,28 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': '§9Gr§de§9nin§dj§9a',
+            'class': 'greninja',
+            'sig': 'ObyIPcEeaSCPynKma2hxkS8S2PWBw1Fb4fPEnKPIXwX41zV+yqH6fcPp8I0s42H2dexzDVuqB+ns+l82+YnKmkbhvxm024n9gZFZdotmd/hyd9Bh35BKgM1ioFZ7YSI8QgDQYQHmQbJLpcvlU46QQTymrQw2ZkM9iB2HLiMtagSv2QjbyCUhNurlYqPr2dew9VXvw9kWPLwkAVAkvyfzPerKIZ+mQMbdl/XOp3k9nf24fFFIOzF4ZBGZZvFUrao0Y+waXvo/QjEuGKwMazcn+61X/dXMrQSk37NSGyKUrx9/92HcZeMvxX9ifiiHCa9bUYJUUGTnv/xxVwlXtG38XGVbfMk1lRNiE4KLdMVkq9BMg8KA6y5aY/lzbi9mQPulR+NCI/ETerDKSlN5ug4g5rUTN5ncEG5TXcs0aHy1g1ZISCIs6oB4SUH102K/1tmfX9yTxChGatyvnJCjax5BnJ63iB6AiZ1m0d9WrF5Zq8T0dYLlKkfE9o+1V5MfwL/OUKDdpRthgdas540j61ZtTqNW8Sx9TNRcmWMZfvAJUDRFiKcBiS8333y2y+DNGjdwBxiRKoZx719luBlNyh4aPLqhyRK1Nt0KWjyS3+3YUXP2AwGR+zoctI/CVggV5gWk3cU1UtrU4XtT5vbrVCc0QpUVy38PX/MhGWBNEad7paY=',
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODc1OTkzMzg1NTAsInByb2ZpbGVJZCI6ImJkNmY2MjIyMGI3NTQ2OTg4YzgzZmExY2EyYWZiMmU5IiwicHJvZmlsZU5hbWUiOiJDaDFja2VuIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9iZjRhYmZmNmMzZGM5MDEyN2YxYWU3Yjc4MTJiZDdjNmRkMTk4ZTEzZjVjOGMzZDViNjQzZWY5ZTI4ZmEwYTAifX19',
+            'trail': '',
+            'op': 'false',
+            'speech': "Gre, greninja!",
+            'powers': array(
+                "/buff",
+                "/jump 0-6",
+                "/run 0-7",
+                "/haste 0-3",                
+                "/invis",
+                "/vis"
```

Comment:
i cannot find or capture video of greninja using this move, but i have a info site: http://pokemondb.net/pokedex/greninja
IK its not video or pics, but no one wants to upload a video of greninja using this move :/